build/cluster.js

```diff
@@ -121,7 +152,7 @@ gulp.task('wait-for-heapster', function(doneFn) {
 
     // constantly query the heapster until it is properly running
     heapsterHealthCheck(function(result) {
-      if (result.length > 0) {
+      if (result === runningState) {
```

Comment:
I fixed heapsterHealthCheck to return correct result not always runningState value